### ISSUE TITLE
Bot repeats same question

### ISSUE THREAD

**Issue by poorni19** [RasaHQ/rasa_core#225]
<Restaurant bot - Bot repeats the same message 2 times.I checked the story file and there is no such flow.not sure why the messages are getting repeated 

**Rasa Core version**:0.8.2

**Python version**: 2.7 anaconda

**Operating system** (windows, osx, ...):Macos

**Issue**: bot conversation
goodbye :(
hi
/Users//anaconda2/lib/python2.7/site-packages/sklearn/preprocessing/label.py:151: DeprecationWarning: The truth value of an empty array is ambiguous. Returning False, but in future this will result in an error. Use `array.size > 0` to check that an array is not empty.
  if diff:
how can I help you?
how can I help you?
I'm on it
what kind of cuisine would you like?
what kind of cuisine would you like?
where?
new york
/Users/anaconda2/lib/python2.7/site-packages/sklearn/preprocessing/label.py:151: DeprecationWarning: The truth value of an empty array is ambiguous. Returning False, but in future this will result in an error. Use `array.size > 0` to check that an array is not empty.
  if diff:
where?
newyork

**Content of domain file** (if used & relevant): same as in restaturant bot 
```yaml

```
<Restaurant bot - Bot repeats the same message 2 times.I checked the story file and there is no such flow.not sure why the messages are getting repeated 

**Rasa Core version**:0.8.2

**Python version**: 2.7 anaconda

**Operating system** (windows, osx, ...):Macos

**Issue**: bot conversation
goodbye :(
hi
/Users//anaconda2/lib/python2.7/site-packages/sklearn/preprocessing/label.py:151: DeprecationWarning: The truth value of an empty array is ambiguous. Returning False, but in future this will result in an error. Use `array.size > 0` to check that an array is not empty.
  if diff:
how can I help you?
how can I help you?
I'm on it
what kind of cuisine would you like?
what kind of cuisine would you like?
where?
new york
/Users/anaconda2/lib/python2.7/site-packages/sklearn/preprocessing/label.py:151: DeprecationWarning: The truth value of an empty array is ambiguous. Returning False, but in future this will result in an error. Use `array.size > 0` to check that an array is not empty.
  if diff:
where?
newyork

**Content of domain file** (if used & relevant): same as in restaturant bot 
```yaml

```

**Comment by sameerkumar16** [comment]
@poorni19 Did you try different python version ?.I think rasa_core works best in python3.6

**Comment by tmbo** [comment]
Can you run the same with `--debug` so that we can take a look at the debug logs?

**Comment by poorni19** [comment]
@sameerkumar16 Yep !! I will try in python 3.6 later on. But this issue does not seem to related to the environment.

**Comment by poorni19** [comment]
@tmbo sure ..will try to re-create the issue. But, is there a way to stop the bot or jump from one flow (story) to another? Bot just keeps on looking for input

**Comment by tmbo** [comment]
the bot is a trained ml model, so it will automatically jump between conversations if it thinks that is appropriate. Always predicting listen is unusual and might be related to an issue with the training data. 
If you want, you can share the whole setup (training data as well as domain specification and commands to reproduce). Until then I will close this issue, as we can not reproduce this.

**Comment by reza-ebrahimi** [comment]
@tmbo I have same problem, I'm trying to create a whole setup.

**Comment by reza-ebrahimi** [comment]
@tmbo I've created a virtual environment for rasa_nlu and rasa_core tests with adding restrauntbot example, you can clone it and produce the problem.

https://github.com/reza-ebrahimi/rasa_venv
